recipes/systemd/meta.yaml

```diff
@@ -0,0 +1,136 @@
+{% set name = "systemd" %}
+{% set version = "243" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/systemd/systemd/archive/v{{ version }}.tar.gz
+  sha256: 0611843c2407f8b125b1b7cb93533bdebd4ccf91c99dffa64ec61556a258c7d1
+  patches:
+    # fixes compilation issue for missing socket decl
+    # from https://bugs.freedesktop.org/show_bug.cgi?id=67075
+    - l2tp.h.patch
+    - netlink.h.patch
+    # adds hook to secure_getenv, if needed
+    - env-util.patch
+    - proc-cmdline.c.patch
+    # adds signal header to get sigemptyset declaration
+    - copy.patch
+    # adds missing definitions
+    - missing_fs.patch
+    - missing_magic.h.patch
+    - missing_prctl.h.patch
+    - missing_fcntl.h.patch
+    - missing_socket.h.patch
+    # adds missing syncfs() implementation
+    - fs-util.c.patch
+    # don't confuse AF_ALG on systems where it is not availble
+    - khash.c.patch
+    # adds missing time definition hooks
+    - time-util.h.patch
+    - time-util.c.patch
+    - sd-event.c.patch
+    # adds missing ioctls definition hooks
+    - terminal-util.c.patch
+    # Adds missing neighbour.h enums and defines
+    - networkd-fdb.h.patch
+    - netlink-types.c.patch
+    # add missing BPF_XOR defines
+    - arp-util.c.patch
+    - dhcp-network.c.patch
+    # add missing K_OFF def
+    - logind-session.c.patch
+    # add missing O_PATH def
+    - btrfs-util.c.patch
+    - chown-recursive.c.patch
+    # add missing types.h
+    - bpf.h.patch
+    # Add missing loop info
+    - loop-util.c.patch
+    - dissect-image.c.patch
+    # adds missing video defs
+    - v4l_id.c.patch
+    # add missing inputs
+    - udev-builtin-input_id.c.patch
+    # fixes for prctl.h
+    - nspawn-oci.c.patch
+    # avoid missing statfs.f_flags
+    - nspawn-patch-uid.c.patch
+    # add missing syscall
+    - machine-dbus.c.patch
+    # add missing capabilities defines
+    - missing_capability.h.patch
+    # missing in.h defines
+    - resolved-dns-scope.c.patch
+    - resolved-dns-stream.c.patch
+    # add missing fcntl.h
+    - socket-proxyd.c.patch
+    - coredump-vacuum.c.patch
+    # avoid clock_adjtime
+    - timesyncd-manager.c.patch
+    # avoid prlimit
+    - nspawn.c.patch
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - m4
+    - meson >=0.49
+    - ninja
+    - pkg-config
+    - coreutils
+  host:
+    - glib
+    - libcap
+    - util-linux >=2.30
+    - libseccomp >=2.3.1
+    - libgcrypt
+    - python >=3.5
+    - libidn2
+    - gnutls >=3.5.3
+    - openssl >=1.1.0
+    - elfutils
+    - gperf
+    - lxml
+    - dbus
+    - xz
+    - libgpg-error
+  run:
+    - python >=3.5
+    - util-linux >=2.30
+    - dbus
+    - glib
+    - xz
+    - elfutils
+    - libgpg-error
+
+test:
+  commands:
+    - test -f "${PREFIX}/lib/libudev.so"
+
+about:
+  home: https://www.freedesktop.org/wiki/Software/systemd/
+  license: LGPL v2.1
```

Comment:
probably good to use the SPDX identifier here

Reply:
Which is?

Reply:
LGPL-2.1

https://spdx.org/licenses/LGPL-2.1.html

Reply:
This would be good as a linter hint :wink:

Reply:
Yes it would! I'm backed up on other things right now but maybe later!

Reply:
Also could you add a fix to the auto-tick bot? This would be great to fix for existing recipes.